puppet-client.html

```diff
@@ -68,10 +68,14 @@
                 border-color: #A11517;
                 color: #FFFFFF;
             }
+            .reload {
+              display: block;
+              cursor: pointer;
+            }
         </style>
         <div class="box-container" hidden$="{{!showReconnectingIn}}"><div class="box notice"><strong>Not connected.</strong> Connecting in {{reconnectionSeconds}}s... <a href="" on-tap="reconnectNow">Retry now</a></div></div>
         <div class="box-container" hidden$="{{!showReconnectingNow}}"><div class="box notice"><strong>Connecting now...</strong></div></div>
-        <div class="box-container" hidden$="{{!showConnectionError}}"><div class="box error"><strong>Connection error.</strong> See console for details. <a href="" on-tap="reload">Reload page</a> </div></div>
+        <div class="box-container" hidden$="{{!showConnectionError}}"><a class="box error reload" on-tap="reload"><strong>Connection error.</strong> See console for details. <u>Click here to reload</u></a></div>
```

Comment:
Please don't use the `u` element. It is not intended for underlined text, but something much weirder: http://html5doctor.com/u-element/

Use `span` instead.